plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -157,6 +160,11 @@ private Stream<DeltaLakeSplit> getSplits(
                         return Stream.empty();
                     }
 
+                    String splitPath = buildSplitPath(tableLocation, addAction);
```

Comment:
`buildSplitPath()` returns encoded path as Iceberg connector, but I didn't touch in this PR.